Makefile

```diff
@@ -116,49 +115,21 @@ gen-diff:  ## (Release) generate the changelog diff between the current schema a
 	sed -i 's/}/&#125;/g' diff.md
 	npx prettier --write diff.md
 
-gen-clean-ts:  ## Remove generated API client for Typescript
-	rm -rf ./${GEN_API_TS}/
-	rm -rf ./web/node_modules/@goauthentik/api/
+gen-client-ts:  ## Build and install the authentik API for Typescript into the authentik UI Application
+	./scripts/gen-client-ts.mjs
 
-gen-clean-go:  ## Remove generated API client for Go
-	rm -rf ./${GEN_API_GO}/
+	npm i --prefix ${GEN_API_TS}
 
-gen-clean-py:  ## Remove generated API client for Python
-	rm -rf ./${GEN_API_PY}/
+	cd ./${GEN_API_TS} && npm link
+	cd ./web && npm link @goauthentik/api
```

Comment:
Fixes issue where locally-generated API package can become out of sync when adding or removing packages in `@goauthentik/web`